cmd/admission/main.go

```diff
@@ -72,7 +73,8 @@ func main() {
 	server := &http.Server{
 		Addr: ":8443",
 		// Require at least TLS12 to satisfy golint G402.
-		TLSConfig: &tls.Config{MinVersion: tls.VersionTLS12, Certificates: []tls.Certificate{certs}},
+		TLSConfig:         &tls.Config{MinVersion: tls.VersionTLS12, Certificates: []tls.Certificate{certs}},
+		ReadHeaderTimeout: 2 * time.Second,
```

Comment:
What is the purpose of this change?

Reply:
There is the finding that the change fixes

```
cmd/admission/main.go:72:13: G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)
	server := &http.Server{
		Addr: ":8443",
		// Require at least TLS12 to satisfy golint G402.
		TLSConfig: &tls.Config{MinVersion: tls.VersionTLS12, Certificates: []tls.Certificate{certs}},
	}
make: *** [Makefile:118: lint] Error 1

```

Reply:
Good catch, that makes sense. I'm not sure what a good value is here, but 2 seconds seems like it might be on the low side. Would 5 seconds make sense?

Reply:
There seems to be an interesting spread here: cluster-api uses 2 seconds, k/k uses 32. I have no objection to 5 seconds.

Reply:
Wow, that is quite the difference. Maybe we should just meet in the middle with something like 16? It does seem unlikely for this to be an issue with any of the suggested values, just don't want this to end up unexpectedly breaking someone.

Reply:
Sounds great. That should be plenty of time.